app/uploaders/attachment_uploader.rb

```diff
@@ -0,0 +1,49 @@
+class AttachmentUploader < CarrierWave::Uploader::Base
+
+  # Include RMagick or MiniMagick support:
+  # include CarrierWave::RMagick
+  # include CarrierWave::MiniMagick
+
+  # Choose what kind of storage to use for this uploader:
+  storage :file
+  storage :fog
```

Comment:
You shouldn't have both. You should use `:file` for development and `:fog` for production